engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/details/subscription-details.controller.js

```diff
@@ -51,5 +51,7 @@ angular.module('Bastion.subscriptions').controller('SubscriptionDetailsControlle
 
             return "";
         };
+
+        $scope.virtWhoToolTip = translate("If the virt-who field is Yes then the subscription requires the use of virt-who. Learn how to configure and use this tool in the <a href=\"https://access.redhat.com/documentation/en/red-hat-satellite/6.2/single/virtual-instances-guide/\">Virtual Instances Guide</a>.");
```

Comment:
as discussed, i've noticed that this shows up for subscriptions that support virt-who but do not require it (i.e. can be used on a hypervisor to provide guest subs, or on a physical host to provide content).   Thus the statement 'requires virt who' is not correct.  I'm not sure how common this is but may be something look in to.